packages/apps/docs/src/components/Search/components/SearchResults.tsx

```diff
@@ -63,30 +75,52 @@ export const SearchResults: FC<IProps> = ({
         <Tabs.Tab value="qa">QA Space</Tabs.Tab>
 
         <Tabs.Content value="docs">
-          <ScrollBox>
-            <ResultCount count={staticSearchResults.length} />
-            <StaticResults
-              limitResults={limitResults}
-              results={staticSearchResults}
-            />
-            {limitResults !== undefined && query !== undefined ? (
-              <Stack justifyContent="flex-end">
-                <Link href={`/search?q=${query}`} passHref legacyBehavior>
-                  <Button
-                    icon={SystemIcons.TrailingIcon}
-                    title="Go to search results"
-                    onClick={clearModal}
-                  >
-                    Go to search results
-                  </Button>
-                </Link>
-              </Stack>
-            ) : null}
+          <ScrollBox disabled={!hasScroll}>
+            {semanticIsLoading && (
+              <LoadingWrapper>
+                <Loading />
+              </LoadingWrapper>
+            )}
+            {semanticError ? (
+              <Notification.Root
+                color={'negative'}
+                expanded={true}
+                icon={SystemIcon.AlertBox}
+              >
+                {semanticError}
+              </Notification.Root>
+            ) : (
+              <>
+                <ResultCount count={semanticResults.length} />
+                <StaticResults
+                  limitResults={limitResults}
+                  results={semanticResults}
+                />
+                {limitResults !== undefined && query !== undefined ? (
```

Comment:
Didn't we have less crazy eslint rules for JSX now? So we can just use `!query`, also rendering `null` when `query` is any falsey value (such as the empty string we are handling in the effect above).